frontend/script.js

```diff
@@ -0,0 +1,57 @@
+function showSideBar() {
+    const sidebar = document.querySelector('.hamburger');
+    sidebar.style.display = 'flex';
+}
+
+function hideSideBar() {
+    const sidebar = document.querySelector('.hamburger');
+    sidebar.style.display = 'none';
+}
+
+// Specify the API endpoint for user data
+const apiUrl = 'http://localhost:8080/products/';
+// Make a GET request using the Fetch API
+fetch(apiUrl)
```

Comment:
Just adding a note for what we talked about yesterday - I would probably wrap the `fetch` invocation in a separate helper, and place that in its own file. 

That way we could reuse the helper any time we'd want to make a request, without having to repeat the `.then`s, `if (!response.ok)` etc. (I know it's only in one place at the moment, but for the exercise of it)

Reply:
fetch now under a separate file. Fetch function is also a function under the Product object